stubs/WTForms/wtforms/fields/simple.pyi

```diff
@@ -0,0 +1,64 @@
+from collections.abc import Callable, Collection, Sequence
+from typing import Any
+from typing_extensions import Self
+
+from wtforms.fields.core import Field, _Filter, _FormT, _Validator, _Widget
+from wtforms.form import BaseForm
+from wtforms.meta import DefaultMeta, _SupportsGettextAndNgettext
+
+class BooleanField(Field):
+    data: bool
+    default: bool | Callable[[], bool] | None
+    false_values: Collection[Any]
+    def __init__(
+        self,
+        label: str | None = None,
+        validators: tuple[_Validator[_FormT, Self], ...] | list[Any] | None = None,
+        false_values: Collection[Any] | None = None,
+        *,
+        filters: Sequence[_Filter] = (),
+        description: str = "",
+        id: str | None = None,
+        default: bool | Callable[[], bool] | None = None,
+        widget: _Widget[Self] | None = None,
+        render_kw: dict[str, Any] | None = None,
+        name: str | None = None,
+        _form: BaseForm | None = None,
+        _prefix: str = "",
+        _translations: _SupportsGettextAndNgettext | None = None,
+        _meta: DefaultMeta | None = None,
+    ) -> None: ...
+
+class StringField(Field):
+    data: str | None
```

Comment:
Would it make sense to make the base Field class generic over the data type?

Reply:
I have considered it, and I think I even tried it out for a bit, but it ultimately seemed too complicated, for too little gain and would probably make matching `Widget` and `Validator` correctly even more of  a pain.